resources/migrations/001_update_migrations.yaml

```diff
@@ -11476,6 +11476,196 @@ databaseChangeLog:
             tableName: metabase_table
             constraintName: idx_unique_table
 
+  - changeSet:
+      id: v55.2025-05-19T16:46:48
```

Comment:
we should be doing v56 now I think... and maybe coordinate with @nvoxland to see if we should wait until his PR is merged before we do a v56 migration?